src/js/components/SearchBar.jsx

```diff
@@ -73,12 +74,12 @@ export class SearchBar extends PureComponent {
       <div>
         {/* search field */}
         <TextField
```

Comment:
Yeah, I think it could be really cool in the next version! Also, nifty highlighting of results would be really fun, which is available to Fuse.js; it's a bit more involved than simply turning it on though.